edx_sga/sga.py

```diff
@@ -200,59 +213,57 @@ def add_xml_to_node(self, node):
                 pass
             else:
                 node.append(child)
-                del node.attrib['solution']
+                del node.attrib["solution"]
 
     @XBlock.json_handler
-    def save_sga(self, data, suffix=''):
+    def save_sga(self, data, suffix=""):
         # pylint: disable=unused-argument,raise-missing-from
         """
         Persist block data when updating settings in studio.
         """
-        self.display_name = data.get('display_name', self.display_name)
+        self.display_name = data.get("display_name", self.display_name)
 
         # Validate points before saving
-        points = data.get('points', self.points)
+        points = data.get("points", self.points)
         # Check that we are an int
         try:
             points = int(points)
         except ValueError:
-            raise JsonHandlerError(400, 'Points must be an integer')
+            raise JsonHandlerError(400, "Points must be an integer")
         # Check that we are positive
         if points < 0:
-            raise JsonHandlerError(400, 'Points must be a positive integer')
+            raise JsonHandlerError(400, "Points must be a positive integer")
         self.points = points
 
         # Validate weight before saving
-        weight = data.get('weight', self.weight)
+        weight = data.get("weight", self.weight)
         # Check that weight is a float.
         if weight:
             try:
                 weight = float(weight)
             except ValueError:
-                raise JsonHandlerError(400, 'Weight must be a decimal number')
+                raise JsonHandlerError(400, "Weight must be a decimal number")
             # Check that we are positive
             if weight < 0:
-                raise JsonHandlerError(
-                    400, 'Weight must be a positive decimal number'
-                )
+                raise JsonHandlerError(400, "Weight must be a positive decimal number")
         self.weight = weight
 
     @XBlock.handler
-    def upload_assignment(self, request, suffix=''):
+    def upload_assignment(self, request, suffix=""):
         # pylint: disable=unused-argument
         """
         Save a students submission file.
         """
         require(self.upload_allowed())
         user = self.get_real_user()
         require(user)
-        upload = request.params['assignment']
+        upload = request.params["assignment"]
         sha1 = get_sha1(upload.file)
         if self.file_size_over_limit(upload.file):
+            size=self.student_upload_max_size()
```

Comment:
it seems wrong can you please revert this ?

Reply:
It's not an error. I have just converted string from `.format` to `fstring` because of the build failure introduced by that error by linter.
`edx_sga/sga.py:253:21: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)`

Here is the reference of the build for you. 
https://github.com/mitodl/edx-sga/runs/3697937074?check_suite_focus=true#step:5:136